Trusted Cluster Labels Docs (#4844)
gravitational/teleport: docs/5.0/trustedclusters.md

auth_service:
  enabled: true
  tokens:

>   # If using static tokens we recommend to use tools like `pwgen -s 32`  

Reviewer: ```suggestion
  # If using static tokens we recommend using tools like `pwgen -s 32`  
```

  # to generate sufficiently random tokens of 32+ byte length
  - trusted_cluster:mk9JgEVqsgz6pSsHf4kJPAHdVDVtpuE0
```

> This token can be used unlimited number of times. 

Reviewer: ```suggestion
This token can be used an unlimited number of times. 
```

### Security Implications

> Consider the security implications when deciding which token method to use.
> Short lived tokens decrease the window for attack but make automation a bit
> more complicated.

Reviewer: (added some detail about the potential complication)

```suggestion
Consider the security implications when deciding which token method to use.
Short lived tokens decrease the window for attack, but will require any automation
which uses these tokens to refresh them on a regular basis.
```

of cluster "root":

```bsh

> # generates a trusted cluster token to allow an inbound from a trusting cluster:

Reviewer: ```suggestion
# generates a trusted cluster token to allow an inbound connect from a leaf cluster:
```

of cluster "root":

```bsh

> # generates a trusted cluster token to allow an inbound connect from a leaf cluster:

Reviewer: ```suggestion
# generates a trusted cluster token to allow an inbound connection from a leaf cluster:
```

  allow:
    node_labels:
      '*': '*'
    # Cluster labels control what clusters user can connect to. The wildcard ('*') means

>     # any cluster. If no role in the role set is using labels and cluster is not labeled,

Reviewer: ```suggestion
    # any cluster. If no role in the role set is using labels and the cluster is not labeled,
```

**NOTE:** The regexp matching is activated only when the expression starts
with `^` and ends with `$`

> 
> 

Reviewer: ```suggestion

```

### Trusted Cluster UI

> For customers using Teleport Enterprise, they can easily configure _leaf_ nodes using the

Reviewer: ```suggestion

For customers using Teleport Enterprise, they can easily configure _leaf_ nodes using the
```

$ tctl create root-user-updated-role.yaml
```

> ### Updating Cluster Labels
> Teleport lets administrator of root clusters the ability to control cluster 
> labels. Letting leaf clusters propagate their labels creates a problem of 
> rogue clusters updating their labels to bad values.

Reviewer: ```suggestion
### Updating Cluster Labels

Teleport gives administrators of root clusters the ability to control cluster labels.
Allowing leaf clusters to propagate their own labels could create a problem with
rogue clusters updating their labels to bad values.
```

> An, administrator of root cluster control the labels
> using remote clusters API without fear of override:

Reviewer: ```suggestion
An administrator of a root cluster can control a remote/leaf cluster's
labels using the remote cluster API without any fear of override:
```

```bash
$ tctl get rc

kind: remote_cluster
metadata:
  name: two
status:
  connection: online
  last_heartbeat: "2020-09-14T03:13:59.35518164Z"
version: v3
```

> Using `tctl` to update the labels on the remote / leaf cluster. 

Reviewer: ```suggestion
Using `tctl` to update the labels on the remote/leaf cluster:
```

```
$ tctl update rc/two --set-labels=env=prod

cluster two has been updated
```

> Using `tctl` to confirm that the updated labels have been set. 

Reviewer: ```suggestion
Using `tctl` to confirm that the updated labels have been set:
```
